feat(docker): Document usage of the Bonita docker image (#1085)
bonitasoft/bonita-doc: md/bonita-bpm-installation-overview.md

For detailed information on the Supported Environment Matrix, see the [Support Guide](https://customer.bonitasoft.com/support-policies) or [ask to be contacted by our commercial team](http://www.bonitasoft.com/contact-us).

## Bonita Docker image
Bonita Platform can be installed in a docker container using the Bonita docker distribution.

> A more detailed look at how to configure and run the container can be found [here](bonita-container.md) 

Reviewer: ```suggestion
Read [how to configure and run the container](bonita-container.md).
```
